site/docs/v1/tech/troubleshooting.adoc

```diff
@@ -142,6 +143,65 @@ If you can still get a lower number of logins per second than you'd expect with
 
 If the MD5 configuration allows you to achieve a much higher logins per second, then the CPU is your bottleneck. If you are CPU bound, the only way to get more logins per second is to horizontally scale (add more nodes) or vertically scale (use larger CPUs with each node).
 
+
+== Troubleshooting Uninstall-Upgrade-Reinstall on RPMs
+
+[WARNING]
+====
+This will completely remove FusionAuth App and FusionAuth Search from your installation. +
+Proceed with caution. It is recommended you back up the database before performing any troubleshooting steps listed here.
+====
+
+=== The Nuclear Option
+
+If you are looking to set up a clean install (scorched earth) using RPM's follow the directions below.
+
+This section also loosely identifies how to properly clean out the FusionAuth directories for a fresh install or upgrade (but this is typically not required unless upgrade issues arise).
+
+The guide below assumes that both FusionAuth App and FusionAuth Search will be uninstalled and reinstalled.
+
+==== RPM - Remove Packages Manually
+
+1. Remove the RPM package for `fusionauth-search` and `fusionauth-app`
+- `sudo rpm -e fusionauth-search`
+- `sudo rpm -e fusionauth-app`
+
+1. Manually delete anything left over from RPM
+- `sudo rm -rf /usr/local/fusionauth/fusionauth-search/`
+- `sudo rm -rf /usr/local/fusionauth/fusionauth-app/`
+
+1. After removal, the following files will remain:
+* `/usr/local/fusionauth/config`
+* `/usr/local/fusionauth/data`
+* `/usr/local/fusionauth/java`
+* `/usr/local/fusionauth/bin`
+** (if it exists)
+* `/usr/local/fusionauth/java`
+** (is also safe to delete, but is not necessary)
+
+1. Run a re-install of FusionAuth App and FusionAuth Search
+* Using the RPM commands
+** `sudo rpm -i <file_to_be_installed>.rpm`
+** link:/direct-download[Direct download packages/rpm link]
+* If on a development machine, FastPath installation is also an option.
+**  link:/docs/v1/tech/installation-guide/fast-path/#fast-path-install[Fast Path Install]
+
+1. This should leave you with a clean install preserving:
```

Comment:
This seems like burying the lede. Move this up to the top, as this is the entire goal, correct?